test/graphql/project_sample_query_test.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class ProjectSampleQueryTest < ActiveSupport::TestCase
+  PROJECT_QUERY = <<~GRAPHQL
+    query($projectPath: ID!) {
+      project(fullPath: $projectPath) {
+        name
+        path
+        id
+        samples {
```

Comment:
**nitpick (blocking):** please add `first: 10` here, or make it a query param and then use that query param, like how $projectPath is used above.